src/main/java/com/google/devtools/build/lib/remote/options/CommonRemoteOptions.java

```diff
@@ -64,6 +68,10 @@ public static class RemoteDurationConverter extends Converter.Contextless<Durati
 
     @Override
     public Duration convert(String input) throws OptionsParsingException {
+      /* We recognize the magic value SERVER_KEYWORD as a way to specify server lifetime ttl. */
+      if (input.equals(SERVER_KEYWORD)) {
```

Comment:
Please move this to a dedicated new converter class, possibly by subclassing this one. The keyword just doesn't make sense for other duration flags such as `--remote_retry_max_delay`.

Reply:
Agreed -- done.